{{cookiecutter.project_slug}}/.tool-versions

```diff
@@ -0,0 +1,3 @@
+# this if config file for https://asdf-vm.com/
```

Comment:
Note that nothing would require use of asdf, but it would provide more convenient flow for those who use it